packages/sdk/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@meeco/sdk",
   "description": "A wrapper library around Meeco's various API SDK's and encryption libraries",
-  "version": "4.20.0",
+  "version": "5.0.0",
```

Comment:
This change doesn't bring any major breaking changes. I suggest `4.21.0` as a new version number.